pkgs/applications/misc/rofi-screenshot/out-directory.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/rofi-screenshot b/rofi-screenshot
```

Comment:
Have you considered upstreaming this patch? It's not nixpkgs/nixos specific so it shouldn't be here imo.

Reply:
Done.

Reply:
Assuming upstream won't be responsive, is it OK to keep this patch as a remote github URL (to the upstream PR) then? Not being able to customize the output directory is a deal-breaker for me to merge this, and while not strictly NixOS-related, it's an important, retro-compatible tweak.

Reply:
Alright, in the meantime upstream did merge the PR. Updating to HEAD and resolving.